gcc/config/loongarch/linux.h

```diff
@@ -21,32 +21,24 @@ along with GCC; see the file COPYING3.  If not see
  * This ensures that a compiler configured with --disable-multilib
  * can work in an multilib environment.  */
 
-#if defined (__DISABLE_MULTILIB) && defined (__DISABLE_MULTIARCH)
-
-  /* Integer ABI  */
-  #if DEFAULT_ABI_INT == ABI_LP64
-    #define INT_ABI_SUFFIX "lib64"
-  #endif
-
-  /* Floating-Point ABI  */
-  #if DEFAULT_ABI_FLOAT == ABI_SOFT_FLOAT
-    #define FLOAT_ABI_SUFFIX "soft/"
-  #elif DEFAULT_ABI_FLOAT == ABI_SINGLE_FLOAT
-    #define FLOAT_ABI_SUFFIX "single/"
+#if defined(__DISABLE_MULTILIB) && defined(__DISABLE_MULTIARCH)
+
+  /* Integer ABI */
+  #if DEFAULT_ABI_INT == ABI_BASE_LP64D
+    #define ABI_SUFFIX "lib64"
+  #elif DEFAULT_ABI_INT == ABI_BASE_LP64F
+    #define ABI_SUFFIX "lib64/f32"
+  #elif DEFAULT_ABI_INT == ABI_BASE_LP64S
+    #define ABI_SUFFIX "lib64/sf"
```

Comment:
Do these defines mean that LIBDIR for LP64F is `/lib64/f32`, and that LIBDIR for LP64S is `/lib64/sf`? I think this should be mentioned in some kind of spec somehow.